crates/autopilot/src/domain/fee/mod.rs

```diff
@@ -4,65 +4,39 @@
 //! we define the way to calculate the protocol fee based on the configuration
 //! parameters.
 
-use crate::{
-    boundary::{self},
-    domain,
+use {
+    crate::{
+        arguments,
+        boundary::{self},
+        domain,
+    },
+    itertools::Itertools,
+    primitive_types::U256,
 };
 
 /// Constructs fee policies based on the current configuration.
-#[derive(Debug)]
 pub struct ProtocolFee {
-    policy: Policy,
-    fee_policy_skip_market_orders: bool,
+    policy_builder: PolicyBuilder,
```

Comment:
Since this is a private type now, if we move it to a `policy` submodule we could even call it `Policy` without having a name collision (the way it turned out now it's not really following the Builder pattern anymore)

Reply:
Moved